rules/windows/builtin/code_integrity/win_codeintegrity_attempted_dll_load.yml

```
@@ -51,6 +51,13 @@ detection:
         ProcessNameBuffer|endswith: '\MsMpEng.exe'
         RequestedPolicy: 7
         ValidatedPolicy: 1
+    filter_avast:
+        FileNameBuffer|endswith:
+            - 'Program Files\Avast Software\Avast\aswAMSI.dll'
+            - 'Program Files (x86)\Avast Software\Avast\aswAMSI.dll'
```

```suggestion
            - 'C:\Program Files\Avast Software\Avast\aswAMSI.dll'
            - 'C:\Program Files (x86)\Avast Software\Avast\aswAMSI.dll'
```

Can we write it like this?

---

No, it's printed using the global object paths. But I stripped a `\` too much.

---

I forgot about the path even though, I worked on the rule 2 days ago 😂.